pkg/whitesource/utilsMock.go

```diff
@@ -42,8 +44,30 @@ func (m *ScanUtilsMock) RemoveAll(_ string) error {
 }
 
 // FindPackageJSONFiles mimics npm.FindPackageJSONFiles() based on the FilesMock setup.
-func (m *ScanUtilsMock) FindPackageJSONFiles(_ *ScanOptions) ([]string, error) {
-	matches, _ := m.Glob("**/package.json")
+func (m *ScanUtilsMock) FindPackageJSONFiles(options *ScanOptions) ([]string, error) {
+	unfilteredMatches, _ := m.Glob("**/package.json")
+	var matches []string
+
+	// Copied from pkg/npm/npm.go
```

Comment:
Please make a comment in `npm.go` too.

Reply:
Can you please have another look? I've moved the file filtering into the piperutils pkg and added tests.